### PR TITLE
ChoiceGroup fluent update

### PR DESCRIPTION
**Pull request by maxwellred**
#### Pull request checklist

- [ ] Addresses an existing issue: #9299
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Updated Image Choice field margin in ChoiceGroup to match toolkit dimensions

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9473)

### CODE REVIEW

**Comment by aneeshack4** [comment]
@maxwellred This is currently a draft PR - are you intending to send it in for review soon?

File: .gitignore [hunk 1]
```diff
@@ -123,3 +123,4 @@ yarn.lock
 
 # Fabric website
 apps/fabric-website/flights/
+common/changes/@uifabric/test-utilities
```
Comment (ecraig12345) [inline .gitignore]: Why did you add this to gitignore?

Comment (maxwellred) [reply .gitignore]: Reverted the mistake, thanks!
